test/unit/support.js

```diff
@@ -56,18 +56,33 @@ testIframe(
 
 ( function() {
 	var expected,
+		version,
 		userAgent = window.navigator.userAgent,
 		expectedMap = {
-			edge: {},
-			ie_11: {},
-			chrome: {},
-			safari: {},
-			firefox: {},
-			ios: {}
+			edge: {
+				responseURL:true
+			},
+			ie_11: {
+				responseURL:false
+			},
+			chrome: {
+				responseURL:true
+			},
+			safari: {
+				responseURL:true
+			},
+			firefox: {
+				responseURL:true
+			},
+			ios: {
+				responseURL:true
+			}
 		};
 
 	if ( /edge\//i.test( userAgent ) ) {
+		version = parseInt( ( /edge\/([0-9.]+)/i ).exec( navigator.userAgent )[ 1 ] );
 		expected = expectedMap.edge;
+		expected.responseURL = version >= 14;
```

Comment:
We only support last 2 Edge versions, currently 17 & 18. This is not needed.

Please revert all changes done to this file.

Reply:
Yes, I didn't know that, fixed now